### PR TITLE
CA Certificates core entity documentations

### PR DESCRIPTION
**Pull request by dndx**

### CODE REVIEW

File: app/1.3.x/db-less-admin-api.md [hunk 1]
```diff
@@ -251,41 +251,69 @@ certificate_data: |
...
         "tags": ["admin", "high-priority", "critical"],
-        "certificate": {"id":"d26761d5-83a4-4f24-ac6c-cff276f2b79c"}
+        "certificate": {"id":"147f5ef0-1ed6-4711-b77f-489262f8bff7"}
     }],
 
 upstream_body: |
     Attributes | Description
     ---:| ---
     `name` | This is a hostname, which must be equal to the `host` of a Service.
+    `algorithm`<br>*optional* | What kinds of load balancing algorithm to use. One of: `round-robin`, `consistent-hashing`, or `least-connections`. Defaults to `"consistent"`.
```
Comment (hishamhm) [inline app/1.3.x/db-less-admin-api.md]: This <tt>Defaults to \`"consistent"\`</tt> is incorrect (default is `round-robin`) and seems to indicate you ran the generator on an outdated pull of `next` (before the latest changes to the `algorithm` field were committed.

File: app/1.3.x/db-less-admin-api.md [hunk 2]
```diff
@@ -1457,15 +1457,15 @@ HTTP 200 OK
 
 ---
 
-### Retrieve Ca_certificate
+### Retrieve Ca Certificate
```
Comment (hishamhm) [inline app/1.3.x/db-less-admin-api.md]: You can add `ca` to the `title_exceptions` array in `autodoc-admin-api/run.lua` so it is properly auto-capitalized :)

File: app/1.3.x/db-less-admin-api.md [hunk 3]
```diff
@@ -1457,15 +1457,15 @@ HTTP 200 OK
...
 
-##### Retrieve Ca_certificate
+##### Retrieve CA Certificate
```
Comment (hishamhm) [inline app/1.3.x/db-less-admin-api.md]: (it's odd that this one didn't get auto-capitalized, looks like a tiny problem in the script)

File: app/1.3.x/db-less-admin-api.md (hunk 2 above)
Comment (dndx) [reply app/1.3.x/db-less-admin-api.md]: Already pushed a commit to fix it.

File: app/1.3.x/db-less-admin-api.md (hunk 1 above)
Comment (dndx) [reply app/1.3.x/db-less-admin-api.md]: Sorry it was indeed outdated. Pushed a new commit.

File: app/1.3.x/db-less-admin-api.md (hunk 2 above)
Comment (hishamhm) [reply app/1.3.x/db-less-admin-api.md]: what I mentioned above is different from the `entity_title` entry

Comment (dndx) [reply app/1.3.x/db-less-admin-api.md]: Got it. That has been added.

**Comment by dndx** [comment]
@kikito Please take a look at the changes to `autodoc-admin-api/run.lua` in https://github.com/Kong/docs.konghq.com/pull/1421/commits/0f014c6ddbde27275cfd174b712308fb4ed5d331. It seems that the original code incorrectly used the field name as entity name and won't work with fields like:

```
{ client_certificate = { type = "foreign", reference = "certificates" }, },
```

as this will cause it to look for core entity `client_certificates` instead of `certificates`. Which causes the following error:

```
ERROR: autodoc-admin-api/run.lua:726: attempt to index local 'feinfo' (a nil value)
stack traceback:
	autodoc-admin-api/run.lua:837: in function 'write_admin_api'
	autodoc-admin-api/run.lua:900: in function 'main'
	autodoc-admin-api/run.lua:922: in function 'file_gen'
	init_worker_by_lua:45: in function <init_worker_by_lua:43>
	[C]: in function 'xpcall'
	init_worker_by_lua:52: in function <init_worker_by_lua:50>
```

File: app/1.3.x/db-less-admin-api.md (hunk 3 above)
Comment (dndx) [reply app/1.3.x/db-less-admin-api.md]: https://github.com/Kong/docs.konghq.com/pull/1421/commits/1473b97b6f217c92df2729e97bcdfcf964f82441 seems to fixed it :)

File: app/1.3.x/admin-api.md [hunk 4]
```diff
@@ -72,6 +76,7 @@ route_body: |
     `methods`<br>*semi-optional* |  A list of HTTP methods that match this Route. When using `http` or `https` protocols, at least one of `hosts`, `paths`, or `methods` must be set. 
     `hosts`<br>*semi-optional* |  A list of domain names that match this Route. When using `http` or `https` protocols, at least one of `hosts`, `paths`, or `methods` must be set.  With form-encoded, the notation is `hosts[]=example.com&hosts[]=foo.test`. With JSON, use an Array.
     `paths`<br>*semi-optional* |  A list of paths that match this Route. When using `http` or `https` protocols, at least one of `hosts`, `paths`, or `methods` must be set.  With form-encoded, the notation is `paths[]=/foo&paths[]=/bar`. With JSON, use an Array.
+    `headers`<br>*optional* | PLACEHOLDER
```
Comment (kikito) [inline app/1.3.x/admin-api.md]: PLACEHOLDER here.

File: app/1.3.x/db-less-admin-api.md [hunk 5]
```diff
@@ -72,6 +76,7 @@ route_body: |
     `methods`<br>*semi-optional* |  A list of HTTP methods that match this Route. When using `http` or `https` protocols, at least one of `hosts`, `paths`, or `methods` must be set. 
     `hosts`<br>*semi-optional* |  A list of domain names that match this Route. When using `http` or `https` protocols, at least one of `hosts`, `paths`, or `methods` must be set.  With form-encoded, the notation is `hosts[]=example.com&hosts[]=foo.test`. With JSON, use an Array.
     `paths`<br>*semi-optional* |  A list of paths that match this Route. When using `http` or `https` protocols, at least one of `hosts`, `paths`, or `methods` must be set.  With form-encoded, the notation is `paths[]=/foo&paths[]=/bar`. With JSON, use an Array.
+    `headers`<br>*optional* | PLACEHOLDER
```
Comment (kikito) [inline app/1.3.x/db-less-admin-api.md]: PLACEHOLDER here.

File: autodoc-admin-api/data.lua [hunk 6]
```diff
@@ -1055,11 +1098,25 @@ return {
         },
 
       },
+
+      ["/upstreams/:upstreams/targets/:targets/:address/healthy"] = {
+        GET = {
+          title = "PLACEHOLDER",
```
Comment (kikito) [inline autodoc-admin-api/data.lua]: PLACEHOLDER here and again down below.

**Comment by dndx** [comment]
@kikito The placeholders are for my changes to render properly. Other people can add their docs later (it will only be in the release branch for now) but no reason to block unrelated docs from being merged because of those.

**Comment by hishamhm** [comment]
Please hold merging this PR as we are reworking the infra for the autogenerated docs. Thank you!

**Comment by dndx** [comment]
No longer needed thanks to the autogen-doc!

**Pull request closed without merging**